revdep/README.md

```diff
@@ -0,0 +1,28 @@
+# Platform
+
+|field    |value                                |
+|:--------|:------------------------------------|
+|version  |R version 4.1.2 (2021-11-01)         |
+|os       |Windows 10 x64 (build 22000)         |
+|system   |x86_64, mingw32                      |
+|ui       |RStudio                              |
+|language |(EN)                                 |
+|collate  |Spanish_Spain.1252                   |
+|ctype    |Spanish_Spain.1252                   |
+|tz       |Europe/Paris                         |
+|date     |2022-01-13                           |
+|rstudio  |2021.09.0+351 Ghost Orchid (desktop) |
+|pandoc   |2.13 @ C:\PROGRA~1\Pandoc\pandoc.exe |
+
+# Dependencies
+
+|package  |old     |new        |<U+0394>  |
+|:--------|:-------|:----------|:--|
+|bibtex   |0.4.2.3 |0.4.3.9000 |*  |
+|glue     |1.6.0   |NA         |*  |
+|magrittr |2.0.1   |NA         |*  |
+|stringi  |1.7.6   |NA         |*  |
+|stringr  |1.4.0   |NA         |*  |
+
```

Comment:
I'm not sure there are any dependencies w.r.t. `stringr` / `stringi` / `glue` / `magrittr`?

Reply:
This one is funny: On CRAN version, `bibtex` imports `stringr` (¿?), that imports `glue`, `magritt` and `stringr`, so I guess thats why they appear on the summary

https://cran.r-project.org/web/packages/bibtex/index.html
https://cran.r-project.org/web/packages/stringr/index.html